package.json

```diff
@@ -35,7 +38,7 @@
         "shelljs": "0.8.2"
     },
     "optionalDependencies": {
-        "gpii-windows": "0.3.0-dev.20200317T211057Z.74cd1fa5"
+        "gpii-windows": "sgithens/windows#GPII-1908"
```

Comment:
Can this be knocked back to the latest gpii-windows release of 0.3.0-dev.20200427T102955Z.2e04c69 since I don't see any pull matching this? Also, a good opportunity to merge up with master

Reply:
There are a few PR's from universal still needed. At the very bare minimum, it requires:
https://github.com/GPII/universal/pull/859

And then realistically for the case studies it also needs the following. One is a very small solutions fix, one is JAWS, and one is Zoomtext.
https://github.com/GPII/universal/pull/857
https://github.com/GPII/universal/pull/846
https://github.com/GPII/universal/pull/735